apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -0,0 +1,696 @@
+                )}
+              />
+            )}
+
+            <div className="px-6">
+              <div className="flex flex-col gap-y-1">
+                <p className="text-sm text-foreground-light">Attachments</p>
+                <p className="text-sm text-foreground-lighter">
+                  Upload up to {MAX_ATTACHMENTS} screenshots that might be relevant to the issue
+                  that you're facing
+                </p>
+              </div>
+              <input
+                multiple
+                type="file"
+                ref={uploadButtonRef}
+                className="hidden"
+                accept="image/png, image/jpeg"
+                onChange={onFilesUpload}
+              />
+              <div className="flex items-center gap-x-2 mt-4">
+                {uploadedDataUrls.map((x: any, idx: number) => (
+                  <div
+                    key={idx}
+                    style={{ backgroundImage: `url("${x}")` }}
+                    className="relative h-14 w-14 rounded bg-cover bg-center bg-no-repeat"
+                  >
+                    <div
+                      className={[
+                        'flex h-4 w-4 items-center justify-center rounded-full bg-red-900',
+                        'absolute -top-1 -right-1 cursor-pointer',
+                      ].join(' ')}
+                      onClick={() => removeUploadedFile(idx)}
+                    >
+                      <X size={12} strokeWidth={2} />
+                    </div>
+                  </div>
+                ))}
+                {uploadedFiles.length < MAX_ATTACHMENTS && (
+                  <div
+                    className={[
+                      'border border-stronger opacity-50 transition hover:opacity-100',
+                      'group flex h-14 w-14 cursor-pointer items-center justify-center rounded',
+                    ].join(' ')}
+                    onClick={() => {
+                      if (uploadButtonRef.current) (uploadButtonRef.current as any).click()
+                    }}
+                  >
+                    <Plus strokeWidth={2} size={20} />
+                  </div>
+                )}
+              </div>
+            </div>
+
+            <div className="px-6">
+              <div className="flex items-center space-x-1 justify-end block text-sm mt-0 mb-2">
+                <p className="text-foreground-light">We will contact you at</p>
+                <p className="text-foreground font-medium">{respondToEmail}</p>
+              </div>
+              <div className="flex items-center space-x-1 justify-end block text-sm mt-0 mb-2">
+                <p className="text-foreground-light">
+                  Please ensure you haven't blocked Hubspot in your emails
+                </p>
+              </div>
+              <div className="flex justify-end">
+                <Button
+                  htmlType="submit"
+                  size="small"
+                  icon={<Mail />}
```

Comment:
Just me, but a Mail icon makes me feel like it's snail mail.

Reply:
i'll keep it as it is for now, won't change it in this PR since this one's focusing on the refactoring part